account_tax_balance/wizard/open_tax_balances.py

```diff
@@ -8,8 +8,11 @@ class WizardOpenTaxBalances(models.TransientModel):
     _name = "wizard.open.tax.balances"
     _description = "Wizard Open Tax Balances"
 
-    company_id = fields.Many2one(
-        "res.company", required=True, default=lambda self: self.env.user.company_id
+    company_ids = fields.Many2many(
+        comodel_name="res.company",
+        string="Companies",
+        required=True,
+        default=lambda self: self.env.company.ids,
```

Comment:
```suggestion
        default=lambda self: self.env.companies.ids,
```
not sure if the variable of allowed companies is that one.